src/app/components/promotions/promotions.component.html

```diff
@@ -0,0 +1,43 @@
+
+<ion-grid class="top-2">
+  <ion-col>
+    <span class="txt-gray-title ml-1">Ofertas del día</span>
+  </ion-col>
+
+  <ion-row class="top-15">
+    <ion-slides [options]="slideOpts">
```

Comment:
hay que remover el ion-slide bro, en ionic 6 se depreco, tenemos que cambiar a swiper
https://ionicframework.com/docs/angular/slides

Reply:
Hola Luis, en esta parte del Slider el elemento que está seleccionado debe ser más grande que los demás elementos, también noté que falta el precio sin descuento y también falta que se llenen las estrellas de acuerdo a las que tenga cada producto.

![image](https://user-images.githubusercontent.com/86495476/169366095-860022ee-bd85-4810-a99e-5480a1273e19.png)